### PR TITLE
zaki/career_page_update

### PR DESCRIPTION
**Pull request by zakibinary**
Updates to career page layout.

### CODE REVIEW

File: src/sass/static/pages.scss [hunk 1]
```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
...
+                    color: $COLOR_WHITE;
+                }
+                &:focus {
+                    border: 0;
+                    outline: 0;
+                }
+            }
+        }
+        .footer-email {
+            background-color: $COLOR_ORANGE;
```
Comment (negarn) [inline src/sass/static/pages.scss]: I think you can use the class `secondary-bg-color` for orange background

File: src/sass/static/pages.scss [hunk 2]
```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
...
+                    max-width: 80%;
+                    font-weight: 300;
+                }
+            }
+        }
+        .margin-top-100 {
+            margin-top: 100px;
+        }
+        .no-margin-top {
+            margin: 0 0 1em;
```
Comment (negarn) [inline src/sass/static/pages.scss]: from the class name it sounds like the style should be `margin-top: 0;`, can we change the style or update the class name?

File: src/sass/static/pages.scss [hunk 3]
```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
...
+                        max-width: 80%;
+                    }
+                }
+            }
+            @media (max-width: 768px) {
+                background-position: 75% -30px;
+            }
+        }
+        .center-banner {
+            background: url('../images/pages/careers/careers-2_bg@3x.jpg') no-repeat;
```
Comment (negarn) [inline src/sass/static/pages.scss]: I suggest we change all the image names to standardize `-` or `_` and remove all the `@3x` etc from it

File: src/sass/static/pages.scss [hunk 4]
```diff
@@ -470,3 +470,213 @@ body #not_authenticated_financial ul.checked > li {
...
+.careers-content {
+    .career {
+        margin-top: 0;
+
+        .top-banner {
+            background: url('../images/pages/careers/careers-1_bg@3x.jpg') no-repeat;
+            background-size: cover;
+            min-height: 300px;
+            position: relative;
+            background-position: center center;
```
Comment (negarn) [inline src/sass/static/pages.scss]: is it the same styling if we emit one `center`? or we need to keep both?

Comment (zakibinary) [reply src/sass/static/pages.scss]: Should be the same if we use only one `center`, i'll update it along with the other fixes.

File: src/templates/static/about/careers.jsx [hunk 5]
```diff
@@ -1,132 +1,268 @@
...
-                <div className='gr-row gr-padding-30 center-text'>
-                    <div className='gr-12 gr-padding-30'>
-                        <h1>{it.L('Our locations')}</h1>
+        <div className='container'>
+            <div className='gr-padding-30 center-text'>
+                <h1>{it.L('Interview process')}</h1>
+            </div>
+            <div className='has-tabs'>
+                <ul className='role-section'>
+                    <li className='role'><a className='role-btn' href='#technical'><span>Technical roles</span></a></li>
```
Comment (binary-hamedanchi) [inline src/templates/static/about/careers.jsx]: Missing localisation?

File: src/templates/static/about/careers.jsx [hunk 6]
```diff
@@ -1,132 +1,268 @@
...
+                    <li className='role'><a className='role-btn' href='#non-technical'><span>Non-technical roles</span></a></li>
```
Comment (binary-hamedanchi) [inline src/templates/static/about/careers.jsx]: Same here.

**Pull request merged**